### PR TITLE
added details of new community folks.

### PR DESCRIPTION
**Pull request by jo-porter**
Fixes https://github.com/hoodiehq/editorial/issues/68
Issue #239, contributors addedto community page and photos for each attached below.

Patricia B&W dsc06685 2
Patricia colour dsc06687
Jen B&W jenn-1
Jen colour jenn_color
Nick B&W nick-herr-bw
Nick colour nick-herr-colour
Christoph colour christoph-witzko
Christoph B&W christoph-witzko-bw

### CODE REVIEW

**Comment by gr2m** [comment]
Hey Jules, the pictures must be added here:
https://github.com/hoodiehq/hoodie-css/tree/gh-pages/src/content_img/community

Could you send a PR for https://github.com/hoodiehq/hoodie-css with the pictures? If not we can do that for you of course. Great work :+1:

**Comment by jo-porter** [comment]
No no, I can do it :) I just didn't know where to add them! 

Thanks :D

**Comment by gr2m** [comment]
I’ve merged your PR for the images. We should scale them down and crop them so that they load faster, but we can do that later. 

For this PR, you got Nick’s last name wrong (It's Hehr, not Herr), and you still need to add the image URLs. I fixed the name of Nick’s image in the hoodie-css repo already :)

Here is the diff for the changes you still need to do before we can merge :) Let us know if you have any questions

``` diff
diff --git a/community/index.html b/community/index.html
index d5d0838..d310df8 100644
--- a/community/index.html
+++ b/community/index.html
@@ -121,8 +121,8 @@ title: hood.ie community
      <div class="cb person-bio">
         <article>
             <figure class="grid-3 l">
-                <img class="bio-img sw" src="" width="152" height="152" alt="Portrait photo(black and white) of Christoph Witzko." />
-                <img class="bio-img bunt" src="" width="152" height="152" alt="Portrait photo(colour) of Christoph Witzko." />
+                <img class="bio-img sw" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/christoph-witzko.jpg" width="152" height="152" alt="Portrait photo(black and white) of Christoph Witzko." />
+                <img class="bio-img bunt" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/hover-bunt/christoph-witzko--hover.jpg" width="152" height="152" alt="Portrait photo(colour) of Christoph Witzko." />
             </figure>
             <span class="h6">Backend Developer &amp; Operations Engineer</span>
             <h1 class="h3">
@@ -352,12 +326,12 @@ title: hood.ie community
     <div class="cb person-bio">
         <article>
             <figure class="grid-3 l">
-                <img class="bio-img sw" src=" " width="152" height="152" alt="Portrait photo(black and white) of Nick Herr." />
-                <img class="bio-img bunt" src=" " width="152" height="152" alt="Portrait photo(colour) of Nick Herr." />
+                <img class="bio-img sw" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/nick-hehr.jpg" width="152" height="152" alt="Portrait photo(black and white) of Nick Hehr." />
+                <img class="bio-img bunt" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/hover-bunt/nick-hehr--hover.jpg" width="152" height="152" alt="Portrait photo(colour) of Nick Hehr." />
             </figure>
             <span class="h6">Frontend Developer &amp; Hoodie Maintainer</span>
             <h1 class="h3">
-              <a name="nick-herr" class="jump-mark">Nick Herr</a>
+              <a name="nick-hehr" class="jump-mark">Nick Hehr</a>
             </h1>
             <p>
                 An empathetic community member, avid Disney parks &amp; animation fan, and bouldering nut. His mustache is a figment of your imagination.
@@ -390,8 +364,8 @@ title: hood.ie community
      <div class="cb person-bio">
         <article>
             <figure class="grid-3 l">
-                <img class="bio-img sw" src="" width="152" height="152" alt="Portrait photo(black and white) of Patricia Garcia." />
-                <img class="bio-img bunt" src="" width="152" height="152" alt="Portrait photo(colour) of Patricia Garcia." />
+                <img class="bio-img sw" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/patricia-garcia.jpg" width="152" height="152" alt="Portrait photo(black and white) of Patricia Garcia." />
+                <img class="bio-img bunt" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/hover-bunt/patricia-garcia--hover.jpg" width="152" height="152" alt="Portrait photo(colour) of Patricia Garcia." />
             </figure>
             <span class="h6">Frontend &amp; Backend Developer</span>
             <h1 class="h3">

```

**Comment by jo-porter** [comment]
Hey Gregor, 

I think that’s all done now :)

**Comment by jennwrites** [comment]
Verified the copy changes, LGTM

**Comment by gr2m** [comment]
- Nick’s Patricia’s, Chris hover images cannot be loaded
- Patricia’s image is too big: http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/patricia-garcia.jpg
- Nick’s image is too big: http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/nick-hehr.jpg
- Jenn’s images are too big and not square: http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/jenn-turner.jpg, http://hoodiehq.github.io/hoodie-css/src/content_img/community/hover-bunt/jenn-turner--hover.jpg
- http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/charlotte-spencer.jpg
- Charlotte’s images are the same, is that by accident? http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/charlotte-spencer.jpg - http://hoodiehq.github.io/hoodie-css/src/content_img/community/hover-bunt/charlotte-spencer--hover.jpg

regarding size, all other images are 150x150px

**Comment by varjmes** [comment]
I believe the images being the same was done by whoever added me when I first became a core member.

**Comment by HipsterBrown** [comment]
Love my bio. Thanks for adding me to the list! :smile:

File: community/index.html [hunk 1]
```diff
@@ -226,6 +316,25 @@ <h1 class="h3">
...
+            <span class="h6">Frontend Developer &amp; Hoodie Maintainer</span>
+            <h1 class="h3">
+              <a name="nick-hehr" class="jump-mark">Nick Hehr</a>
+            </h1>
+            <p>
+                An empathetic community member, avid Disney parks &amp; animation fan, and bouldering nut. His mustache is a figment of your imagination.
+            </p>
+        </article>
+        <aside class="bio-link">
+            <a href="https://github.com/hipsterbrown" target="_blank">GitHub</a>
```
Comment (HipsterBrown) [inline community/index.html:331]: Feel free to add my Twitter as well. 

```
<a href="https://twitter.com/hipsterbrown" target="_blank">Twitter</a>
```

**Comment by gr2m** [comment]
merged via 9ebaf61. Thanks so much for your help & patience with us @jo-porter

**Pull request closed without merging**